### PR TITLE
[_]: fix/testimonials-text-fixed

### PR DESCRIPTION
**Pull request by xabg2**

### CODE REVIEW

File: components/home/TestimonialsSection.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import Link from 'next/link';
+
+const TestimonialsSection = ({ textContent }) => {
+  const data = textContent.cards;
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const beforeIndex =
+    currentIndex !== 0 ? textContent.cards.slice(0, currentIndex) : textContent.cards.slice(4, data.length);
+  const current =
+    // currentIndex === 5
```
Comment (PixoDev) [inline components/home/TestimonialsSection.js]: Comments without explanations should be removed if they don't help to understand how the code works

File: components/home/TestimonialsSection.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+            <div
+              key={index}
+              className="card-soft relative flex h-[339px] w-[465px] shrink-0 flex-col rounded-3xl bg-white p-10"
+            >
+              <div className="flex flex-row">
+                <img src="/images/home/testimonials/Comas.svg" />
+                <div className="flex flex-col pl-4">
+                  <p className="text-xl font-semibold">{card.name}</p>
+                  {card.name.includes('Eva') ? (
+                    <Link href={'https://fixthephoto.com/internxt-review.html'}>
```
Comment (PixoDev) [inline components/home/TestimonialsSection.js:75]: Whats this URL?

File: components/layout/BFBanner.js [hunk 3]
```diff
@@ -25,6 +24,9 @@ const BFBanner = () => {
 
   React.useEffect(() => {
     const hideBanner = localStorage.getItem('hideBanner');
+    if (showBanner) {
+      window.dispatchEvent(new Event('CloseSquare'));
+    }
     if (hideBanner === 'true') {
```
Comment (PixoDev) [inline components/layout/BFBanner.js]: better cast this:

`if(Boolean(hideBanner) === true)`

File: components/layout/LoginBanner.js [hunk 4]
```diff
@@ -48,7 +47,10 @@ const BFBanner = () => {
 
   React.useEffect(() => {
     const hideBanner = localStorage.getItem('hideLoginBanner');
-    if (hideBanner === 'true') {
+    if (showBanner) {
```
Comment (PixoDev) [inline components/layout/LoginBanner.js:49]: showBanner and hideBanner looks like the same variable but inverted `hideBanner` is equal to `!showBanner`?

**Review by PixoDev — CHANGES_REQUESTED** [review]

File: components/home/TestimonialsSection.js (hunk 2 above)
Comment (xabg2) [inline components/home/TestimonialsSection.js:72]: redirection to fixthephoto.com by marketing request

**Review by PixoDev — APPROVED** [review]

**Pull request merged**